src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,7 @@
+body {
```

Comment:
Resetting is setting default browser styles like for example margins to value of 0, here you only did that for the body. Generally it would be best to use something like the [Meyer Reset](https://meyerweb.com/eric/tools/css/reset/)